Gemfile

```diff
@@ -30,6 +30,8 @@ gem 'pmp', '~> 0.5'
 
 gem 'hyperresource'
 
+gem 'addressable'
```

Comment:
I find it hard to imagine a feed relying on a fragment, but I guess anything is possible.